### PR TITLE
added check if directory exists

### PR DESCRIPTION
**Pull request by ich777**
I think this should do the job if your plugin is installed first.

### CODE REVIEW

File: gpustat.plg [hunk 1]
```diff
@@ -98,6 +98,12 @@
     elseif (`which intel_gpu_top`) {
         echo(sprintf($banner_out, "Intel vendor utility found. Continuing install."));
     }
+    elseif (`ls /boot/config/plugins/nvidia-driver`) {
+        echo(sprintf($banner_out, "NVIDIA vendor utility found. Continuing install."));
+    }
+    elseif (`ls /boot/config/plugins/intel-gpu-top`) {
+        echo(sprintf($banner_out, "Intel vendor utility found. Continuing install."));
+    }
```
Comment (b3rs3rk) [inline gpustat.plg:106]: If you can, do a new commit and just change my existing if/elseif stanzas with the which commands to add a file_exists check:

```php
    if (`which nvidia-smi` || file_exists('/boot/config/plugins/nvidia-driver')) {
        echo(sprintf($banner_out, "NVIDIA vendor utility found. Continuing install."));
    }
    elseif (`which intel_gpu_top` || file_exists('/boot/config/plugins/intel-gpu-top')) {
        echo(sprintf($banner_out, "Intel vendor utility found. Continuing install."));
    }
```
I'm a little anal about the code, and I don't like duplicating anything, especially echo output strings if I don't have to.

**Review by b3rs3rk — CHANGES_REQUESTED** [review]
Make requested changes and I'll approve

File: gpustat.plg (hunk 1 above)
Comment (ich777) [reply gpustat.plg]: Changes made.

Comment (b3rs3rk) [reply gpustat.plg]: Perfect.

**Review by b3rs3rk — APPROVED** [review]

**Pull request merged**